utilities/adapter-registry.js

```diff
@@ -33,7 +33,7 @@ module.exports = CoreObject.extend({
   },
 
   _mergeDeployAdapters: function(addon) {
-    if (addon.type === 'ember-deploy-addon') {
+    if (addon.type === 'ember-cli-deploy-addon') {
```

Comment:
This is a breaking change to plugins, and we plan a different breaking API change (require package.json keywords to include "ember-cli-deploy-plugin" and implement hooks instead of adapters hash). Maybe we should wait to break the plugins until we have the planned API ready for them?